src/OpenTelemetry.Api/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,327 @@
+abstract OpenTelemetry.Metrics.CounterMetric<T>.Bind(OpenTelemetry.Metrics.LabelSet labelset) -> OpenTelemetry.Metrics.BoundCounterMetric<T>
+abstract OpenTelemetry.Metrics.CounterMetric<T>.Bind(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> labels) -> OpenTelemetry.Metrics.BoundCounterMetric<T>
+abstract OpenTelemetry.Metrics.DoubleObserverMetric.Observe(double value, OpenTelemetry.Metrics.LabelSet labelset) -> void
+abstract OpenTelemetry.Metrics.DoubleObserverMetric.Observe(double value, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> labels) -> void
+abstract OpenTelemetry.Metrics.DoubleObserverMetricHandle.Observe(double value) -> void
+abstract OpenTelemetry.Metrics.Int64ObserverMetric.Observe(long value, OpenTelemetry.Metrics.LabelSet labelset) -> void
+abstract OpenTelemetry.Metrics.Int64ObserverMetric.Observe(long value, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> labels) -> void
+abstract OpenTelemetry.Metrics.Int64ObserverMetricHandle.Observe(long value) -> void
+abstract OpenTelemetry.Metrics.MeasureMetric<T>.Bind(OpenTelemetry.Metrics.LabelSet labelset) -> OpenTelemetry.Metrics.BoundMeasureMetric<T>
+abstract OpenTelemetry.Metrics.MeasureMetric<T>.Bind(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> labels) -> OpenTelemetry.Metrics.BoundMeasureMetric<T>
+abstract OpenTelemetry.Metrics.Meter.CreateDoubleCounter(string name, bool monotonic = true) -> OpenTelemetry.Metrics.CounterMetric<double>
+abstract OpenTelemetry.Metrics.Meter.CreateDoubleMeasure(string name, bool absolute = true) -> OpenTelemetry.Metrics.MeasureMetric<double>
+abstract OpenTelemetry.Metrics.Meter.CreateDoubleObserver(string name, System.Action<OpenTelemetry.Metrics.DoubleObserverMetric> callback, bool absolute = true) -> OpenTelemetry.Metrics.DoubleObserverMetric
+abstract OpenTelemetry.Metrics.Meter.CreateInt64Counter(string name, bool monotonic = true) -> OpenTelemetry.Metrics.CounterMetric<long>
+abstract OpenTelemetry.Metrics.Meter.CreateInt64Measure(string name, bool absolute = true) -> OpenTelemetry.Metrics.MeasureMetric<long>
+abstract OpenTelemetry.Metrics.Meter.CreateInt64Observer(string name, System.Action<OpenTelemetry.Metrics.Int64ObserverMetric> callback, bool absolute = true) -> OpenTelemetry.Metrics.Int64ObserverMetric
+abstract OpenTelemetry.Metrics.Meter.GetLabelSet(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, string>> labels) -> OpenTelemetry.Metrics.LabelSet
+const OpenTelemetry.Trace.SpanAttributeConstants.DatabaseStatementTypeKey = "db.statement_type" -> string
+const OpenTelemetry.Trace.SpanAttributeConstants.HttpPathKey = "http.path" -> string
+const OpenTelemetry.Trace.SpanAttributeConstants.StatusCodeKey = "otel.status_code" -> string
```

Comment:
do we need to make this public if we have an extension method? Can we ask everybody use extension methods instead?

Reply:
Just trying to minimize the public surface

Reply:
we removed in the past few prs. thanks for reviewing it!